src/Fabulous/Array.fs

```diff
@@ -61,7 +61,7 @@ module Array =
     let inline sortInPlace<'T, 'V when 'V: comparison> ([<InlineIfLambda>] getKey: 'T -> 'V) (attrs: 'T[]) : 'T[] =
         let N = attrs.GetLength(0)
 
-        for i in [ 1 .. N - 1 ] do
+        for i in 1 .. N - 1 do
```

Comment:
This is very important -
https://sharplab.io/#v2:DYLgZgzgPsCmAuACAHogXogvAWAFCIMTAHsAnRAS0oDtEBGRAOkfoAZEATYvQ39SnnwJ44SAJ78c+QiXJUKtANr0mLOuwC6nbtKEYKQA